crates/ruff_linter/src/rules/flake8_boolean_trap/mod.rs

```diff
@@ -44,4 +46,25 @@ mod tests {
         assert_messages!(snapshot, diagnostics);
         Ok(())
     }
+
+    #[test]
+    fn extend_allowed_callable() -> Result<()> {
+        let diagnostics = test_path(
+            Path::new("flake8_boolean_trap/FBT.py"),
+            &LinterSettings {
+                flake8_boolean_trap: super::settings::Settings {
+                    extend_allowed_calls: vec![
+                        "Value".to_string(),
+                        "Field".to_string(),
+                        "settings".to_string(),
+                        "deploy".to_string(),
+                        "used".to_string(),
```

Comment:
It's not clear to me if these should be function names, or symbol paths (like `["pydantic", "BaseModel"]`).

Reply:
Do you mean it's not clear in the documentation/implementation? Or are you saying your not sure how we should implement it?

Reply:
The latter!

Reply:
I think symbol path gives finer control, and is less prone to false negatives. The only downside is that it is less intuitive to users, and may lead to confusion. But I think this should be ok with good documentation.

Reply:
I will change it to symbol paths if everyone agrees.

Reply:
is it hard to support both?

Reply:
Symbol path is a little problematic because it doesn't support _methods_ (e.g., there's no way to identify `y` in `x = pydantic.BaseModel(); x.y()`), and methods are a common use-case here.